internal/gatewayapi/testdata/httproute-with-mirror-filter-service-not-found.out.yaml

```diff
@@ -72,9 +72,10 @@ httpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: Route is accepted
-        reason: Accepted
-        status: "True"
+        message: 'Backend service validation failed: Service default/service-unknown
```

Comment:
this 2 similar conditions here
```
Backend service validation failed: Service default/service-unknown
```

&&


```
 Service default/service-unknown not found
```

Reply:
Would it be better to provide a static message explaining why the status is not accepted, such as “invalid configuration detected”? This approach would avoid repeating the message in the accepted status reason. The types, such as “Accepted” and “ResolvedRefs,” are already distinct.

Reply:
I think its fine for this PR